packages/component-library/src/components/form/mt-button/mt-button.vue

```diff
@@ -22,7 +22,13 @@
   >
     <mt-loader v-if="isLoading" size="16px" class="mt-button__loader" />
     <span class="mt-button__content" :class="contentVisibilityClass">
-      <slot />
+      <span v-if="$slots.iconFront" class="mt-button__icon mt-button__icon--front">
```

Comment:
The extra classes like `mt-button__icon--front` is not used anywhere.

Reply:
I've also removed this